Adyen/Helpers/URLSessionHelpers.swift

```diff
@@ -1,5 +1,5 @@
 //
-// Copyright (c) 2019 Adyen N.V.
+// Copyright (c) 2020 Adyen N.V.
```

Comment:
We need to ask IS fro swift-format again 😅

Reply:
@joostvandijk